internal/connector/pingone_platform/resources/pingone_agreement_localization.go

```diff
@@ -0,0 +1,83 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/patrickcping/pingone-go-sdk-v2/pingone"
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+// Verify that the resource satisfies the exportable resource interface
+var (
+	_ connector.ExportableResource = &PingoneAgreementEnableResource{}
+)
+
+type PingoneAgreementLocalizationResource struct {
+	context       context.Context
```

Comment:
Maybe we ought to just have a separate struct for these values since we need the same thing in all of our resources.

Reply:
Added!